cadvisor.go

```diff
@@ -35,6 +36,7 @@ import (
 	"github.com/golang/glog"
 )
 
+var argPath = flag.String("listen_path", "", "Path to listen on (UNIX socket), defaults to empty (use TCP instead)")
```

Comment:
Sure. I'm out next week, but I can take fare of this when I come back.

Do we want backwards compatibility with the existing port flag?

On Friday, 22 April 2016, Tim St. Clair <notifications@github.com> wrote:

> In cadvisor.go
> <https://github.com/google/cadvisor/pull/1225#discussion_r60780849>:
>
> > @@ -35,6 +36,7 @@ import (
> >  	"github.com/golang/glog"
> >  )
> >
> > +var argPath = flag.String("listen_path", "", "Path to listen on (UNIX socket), defaults to empty (use TCP instead)")
>
> Removing flags is a lot harder than adding them, so if we want to use a
> --listen flag, I'd prefer not to add a --listen_path first.
>
> In other words, yes, would you mind adding it in this patch?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/google/cadvisor/pull/1225/files/7fc1a906253f727cee5b662c05153929cbfc8b43#r60780849>
>